.cspell.json

```diff
@@ -19,6 +19,7 @@
       "noSuggest": true
     }
   ],
+  "words": ["OpenAPI"],
```

Comment:
We have a dedicated `project-words.txt` for this. You will also need to transform it to lowercase when you add it there